apps/vx-scan/backend/package.json

```diff
@@ -15,9 +15,7 @@
     "lint": "pnpm type-check && eslint .",
     "lint:fix": "pnpm type-check && eslint . --fix",
     "pre-commit": "lint-staged",
-    "start": "node ./build/index.js",
-    "start:central": "VX_MACHINE_TYPE=bsd node ./build/index.js",
-    "start:precinct": "VX_MACHINE_TYPE=precinct-scanner node ./build/index.js",
+    "start": "VX_MACHINE_TYPE=precinct-scanner node ./build/index.js",
```

Comment:
we should remove the need for this env var when we remove the central scan code from this subdir